aws/inputseeders/aws/masterdata.json

```diff
@@ -1356,18 +1356,18 @@
             "Priority": 10,
             "DistrictType" : "account",
             "ResourceSets": {
-                "cmk" : {
-                    "deployment:Unit" : "cmk",
-                    "deployment:Priority" : 20,
-                    "GroupDeploymentUnit" : false
-                },
                 "iam": {
                     "deployment:Unit": "iam",
-                    "deployment:Priority" : 30,
+                    "deployment:Priority" : 10,
                     "ResourceLabels": [
                         "iam"
                     ]
                 },
+                "cmk" : {
```

Comment:
We have traditionally created the cmk as the first thing for a segment. What drove the switch in ordering?

Reply:
The CMK grants access to ServiceRoles, in order to do that the service roles have to exist or the CMK will fail its creation. 

Its weird a resource policy that actually checks that the principals exist that you are trying to use